pkg/plugin/plugin.go

```diff
@@ -367,6 +367,15 @@ func CmdAdd(args *skel.CmdArgs) error {
 		}
 		result = newResult
 		result.Interfaces = []*current.Interface{hostIface, result.Interfaces[0]}
+
+		for ifIndex, ifCfg := range result.Interfaces {
+			// Adjust interface index with new container interface index in result.Interfaces
+			if ifCfg.Name == args.IfName {
+				for ipIndex, _ := range result.IPs {
+					result.IPs[ipIndex].Interface = current.Int(ifIndex)
```

Comment:
can't we just hardcode ifIndex into 1 (as we knew containerIface is 2nd element in the result.Interfaces ? 
This would avoid outer for and if blocks.

Reply:
I could do that. I usually write quite defensive code, the outer loops are there to protect against an eventual re-ordering of interfaces in the result.

Reply:
The result.Interfaces is just created locally before this block, an array and ordering is guaranteed (as per insertion of elements), we can just believe on this invariant.  But I just leave it to you to decide on the approach.